src/Event.ts

```diff
@@ -0,0 +1,214 @@
+'use strict';
+
+import pino from 'pino';
+
+require('isomorphic-fetch');
+import pkg from '../package.json';
+
+const UA = `Node/${pkg.version}`;
```

Comment:
Use require to make comman `npm run doc` available

const pkg = require('./package.json');